main.tf

```diff
@@ -154,7 +154,8 @@ module "gke_app" {
   local_restore    = var.local_restore
   other_wandb_env = merge({
     "GORILLA_DISABLE_CODE_SAVING"          = var.disable_code_saving,
-    "GORILLA_CUSTOMER_SECRET_STORE_SOURCE" = local.secret_store_source
+    "GORILLA_CUSTOMER_SECRET_STORE_SOURCE" = local.secret_store_source,
+    "GORILLA_GLUE_LIST"                    = var.enable_operator
```

Comment:
when `GORILLA_GLUE_LIST` is true, it _disables_ glue tasks for that pod. So if operator feature flag is on, disable glue task execution in old pod